### DISCUSSION TITLE
Update display firmware / probe_offset_wizard

### DISCUSSION THREAD

**Discussion by galluccib** [Jyers/Marlin#573] [Q&A] [answered]
Noticed probe_offset_wizard is not set by default and was going to enable and compile but not sure if the new display firmware supports it.

Thanks,

**Comment by Jyers** [comment, marked as answer]
I didn't tie into the define for the firmware. I assumed if you have a z offset you'd want a wizard. No need to worry about the define.

**Reply by galluccib** [reply]
I was going to compile and add... I was just wondering will the new display firmware support it.

**Reply by Jyers** [reply]
Yes, it is built in to the firmware. No need to compile to enable it. Its in Prepare>Z-Offset

**Reply by galluccib** [reply]
My bad... noticed the configs it was not enabled so figured it wasn't in the firmware...

#if HAS_BED_PROBE
    //#define PROBE_OFFSET_WIZARD
